docs/features/jsonSchema.md

```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
 The new JSON Schema is structured to validate the YAML files that are loaded via JCasC.
 The structure and validation of the Schema is done based on the user-installed plugins.
 
-=== Version
+ ***Version***
 
 The Schema uses JSON draft v07. 
 http://json-schema.org/draft-07/schema#
 
-=== How to use
+***Using JSON Schema***
 
 * The schema will be available at /configuration-as-code/schema
 * Users can use various online JSON validators to check against their YAML/json.
 
-=== Progress
+***Progress***
 
 * The new JSON Schema is partially working and is in beta mode.
 * The schema validates any missing  configurator objects and invalid data formats.
 * It is built around the existing executable xml (jelly files) and follows the latest draft version of JSON.
 * The schema does not yet validate against deep nested configurations.
 * We are working towards supporting deep nested YAML files.
+* The project has improved Jenkins configuration as code development tools through a CommunityBridge project.
 
-== Improvements
+***Improvements***
 
 * The Old Schema was generated using Jelly files that did not generate a valid JSON Schema.
 * Without a valid schema it is not possible to validate a YAML file rendering the schema unusable.
 * The previous schema included an `"$ref":` for an `object` type, the new draft of the schema makes it mandatory
   to use `"$id":` hence the new schema uses it instead of ref.
-
-== What the schema does
+* The new improvement on [DEV-Tools](https://www.jenkins.io/projects/jcasc/dev-tools/) will be used as an experiment with CommunityBridge and as a project to improve development tools for Jenkins configuration as a code.
+
+ *Issues* 
```

Comment:
This all renders a bit weird, https://github.com/Chinmay922/configuration-as-code-plugin/blob/master/docs/features/jsonSchema.md

I don't think italics are needed, either sub headings or bold would be best imo